code/game/objects/items/storage/holsters.dm

```diff
@@ -0,0 +1,255 @@
+///Parent item for all holster type storage items
+
+/obj/item/storage/holster
+	name = "holster"
+	desc = "Holds stuff, and sometimes goes swoosh."
+	icon_state = "backpack"
+	w_class = WEIGHT_CLASS_BULKY
+	max_w_class = 4 ///normally the special item will be larger than what should fit. Child items will have lower limits and an override
+	storage_slots = 1
+	max_storage_space = 4
+	flags_equip_slot = ITEM_SLOT_BACK
+	draw_mode = 1
+	allow_drawing_method = TRUE
+	///is used to store the 'empty' sprite name
+	var/base_icon = "m37_holster"
+	///the sound produced when the special item is drawn
+	var/drawSound = 'sound/weapons/guns/misc/rifle_draw.ogg'
```

Comment:
Should be draw_sound. We do not do DrawSound on this codebase

Reply:
>mfw I shitcode by trying not to shitcode by copying existing code. But it is also shitcode.